### PR TITLE
Added funding acknowledgements

### PR DESCRIPTION
**Pull request by jdgarciauc3m**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,3 +39,9 @@ Please cite this publication in any work using our library:
 
 * **A C++ Generic Parallel Pattern Interface for Stream Processing**. David Del Río Astorga, Manuel F. Dolz, Luis Miguel Sanchez, Javier Garcia Blas and J. Daniel Garcia. *16th International Conference on Algorithms and Architectures for Parallel Processing (ICA3PP)*. Granada, Spain. 14-16/12/2016. pp. 74-84. DOI: [10.1007/978-3-319-49583-5_5](http://dx.doi.org/10.1007/978-3-319-49583-5_5)
 
+### Acknowledgments
+
+The **GrPPI** library has been partially supproted by:
```
Comment (mdolz) [inline README.md]: Replace `supproted` by `supported`.

Please, consider the use of `ispell` before committing any document to the repository.

File: README.md [hunk 2]
```diff
@@ -38,3 +39,9 @@ Please cite this publication in any work using our library:
...
+
+* Project ICT 644235 *"REPHRASE: REfactoring Parallel Heterogeneous Resource-aware Applications"* funded by the European Comision through H2020 program (2015-2018).
```
Comment (mdolz) [inline README.md]: Replace `comision` by `commission`.

Please, consider the use of `ispell` before committing any document to the repository.

File: README.md [hunk 3]
```diff
@@ -38,3 +39,9 @@ Please cite this publication in any work using our library:
...
+* Project TIN2016-79673-P *“Towards Unification of HPC and Big Data Paradigms”* funded by the Spanish Ministry of Economy and Comeptitiveness (2016-2019).
```
Comment (mdolz) [inline README.md]: Replace `Comeptitiveness` by `Competitiveness`.

Please, consider the use of `ispell` before committing any document to the repository.

**Review by mdolz — CHANGES_REQUESTED** [review]
See comments.

**Comment by jdgarciauc3m** [comment]
I commited a new version

**Review by javiergarcia — APPROVED** [review]
I approve.

**Review by mdolz — APPROVED** [review]

**Review by Drioastorga — APPROVED** [review]

**Review by javidoc00 — APPROVED** [review]
OK

**Pull request merged**